app/models/item.rb

```diff
@@ -6,4 +6,11 @@ class Item < ActiveRecord::Base
   validates :name, presence: true, length: { maximum: 200 }
   validates :quantity, presence: true, numericality: true
   validates :organization, presence: true
+
+  include PgSearch
+    pg_search_scope :search,
+      :against => :name,
+      :using => {
+        :tsearch => {:prefix => true}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.